pootle/apps/pootle_store/migrations/0016_denormalize_project.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def denormalize_project(apps, schema_editor):
+    Unit = apps.get_model("pootle_store", "Unit")
+    Store = apps.get_model("pootle_store", "Store")
+    stores = Store.objects.values_list(
+        "pk", "translation_project__project_id")
+
+    for store_id, project_id in stores.iterator():
+        if not project_id:
```

Comment:
Is this a real possibility? If yes we might end with units without project. See https://github.com/translate/pootle/pull/4527/files#r54554808 below.